src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimantClaimMatcher.java

```diff
@@ -77,19 +77,36 @@ public boolean hasResponseFullAdmit() {
 
     @Override
     public boolean defendantRespondedWithFullAdmitAndPayImmediately() {
+        if (isClaimProceedInCaseMan()) {
+            return false;
+        }
         return hasResponseFullAdmit()
             && isPayImmediately();
     }
 
+    private boolean isClaimProceedInCaseMan() {
+        return featureToggleService.isLipVLipEnabled()
+                && Objects.nonNull(caseData.getTakenOfflineDate())
+                && Objects.nonNull(caseData.getPreviousCCDState())
+                && (caseData.getPreviousCCDState().equals(CaseState.AWAITING_APPLICANT_INTENTION)
```

Comment:
why you are checking the previous ccd state ?  'case proceeds in case man' can be submitted at any point

Reply:
As per ticket CIV-13495, new dashboard status applicable to these two states